third_party/smol-v/tnt/CMakeLists.txt

```diff
@@ -13,10 +13,13 @@ set(SRCS
 add_library(${TARGET} STATIC ${SRCS})
 
 target_compile_options(${TARGET} PRIVATE
-        -Wno-tautological-unsigned-enum-zero-compare
         $<$<PLATFORM_ID:Linux>:-fPIC>
 )
 
+if (NOT MSVC)
+    target_compile_options(${TARGET} PRIVATE Wno-tautological-unsigned-enum-zero-compare)
```

Comment:
I think this needs a dash prepended:`-Wno-tautological-unsigned-enum-zero-compare`

Reply:
While it probably should have a dash, I don't think this is supported in MSVC.

Reply:
I added the dash because it looked like it cause a failure of the builds below.